website/content/docs/services/discovery/dns-forwarding/index.mdx

```diff
@@ -0,0 +1,182 @@
+---
+layout: docs
+page_title: DNS forwarding
+description: ->
+  Learn how to configure your local DNS servers to perform DNS forwarding to Consul servers.
+---
+
+# DNS forwarding
+
+This topic describes the process to configure different DNS servers to enable DNS forwarding to Consul servers.
+
+You can apply these operations on every node where a Consul agent is running.
+
+## Introduction
+
+You deployed a Consul datacenter and want to use Consul DNS interface for name resolution.
+
+When configured with default values, Consul exposes the DNS interface on port `8600`. By default, DNS is served from port `53`. On most operating systems, this requires elevated privileges. It is also common, for most operating systems, to have a local DNS server already running on port `53`.
+
+Instead of running Consul with an administrative or root account, you can forward appropriate queries to Consul, running on an unprivileged port, from another DNS server or using port redirect.
+
+There are two possible configurations for DNS forwarding:
+ - **Conditional DNS forwarding**: the local DNS servers are configured to forward to Consul only queries relative to the `.consul` zone. All other queries are still served via the default DNS server in the node.
+ - **DNS forwarding**: Consul serves all DNS queries and forwards to a remote DNS server the ones outside `.consul` domain.
+
+### Conditional DNS forwarding
+
+We recommend the conditional DNS forwarding approach. 
+
+![Consul DNS conditional forwarding - Only .consul requests are routed to Consul](/img/consul-dns-conditional-forwarding.png)
+
+In this configuration, Consul only serves queries relative to the `.consul` domain. There is no unnecessary load on Consul servers to serve queries from different domains.
+
+### DNS forwarding
+
+This approach can be useful in scenarios where the Consul agent nodes have low resources, and you want to avoid the overhead of running a local DNS server. In this configuration, Consul serves all DNS queries for all domains and forwards the ones outside the `.consul` domain to one or more configured forwarder servers.
```

Comment:
you meant low resource requirements? if the consul nodes have low resources, my hunch would be not to overload them with all the dns traffic of the environment

Reply:
Thanks for the comment. Ideally a node with few resources would not do much DNS traffic except for the one necessary to locate/resolve other services in the DC. In this scenario having to install a local DNS server to act as a forwarding for `.consul` queries is a bit of an overkill.  So this means we can redirect all the traffic directly to Consul knowing that there's not much DNS traffic needed.

Reply:
ohhh i see, you meant low Consul-related workload resources! i assumed it was instance resources like CPU and memory that were referred to 😅 I completely agree with the statement then